core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/workspacemanager/HttpWorkspaceManagerDAO.scala

```diff
@@ -250,4 +250,7 @@ class HttpWorkspaceManagerDAO(apiClientProvider: WorkspaceManagerApiClientProvid
       landingZoneId
     )
   }
+
+  override def getStatus(): Unit =
```

Comment:
`
```suggestion
  override def getStatus(): SystemStatus =
```

Reply:
I used `Unit` because `apiClientProvider.getUnauthenticatedApi().serviceStatus()` in line 255 is a void method (I get a compiler error when I change it to SystemStatus.)

Reply:
`getStatus` returning `Unit` is weird... Maybe rename it to something more descriptive like `throwWhenUnavailable`